.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+on: [push, pull_request, workflow_dispatch] #workflow_dispatch works only if its active in the main branch
+jobs:
+  Unit-Test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setting up JDK 12
+      uses: actions/setup-java@v2
+      with:
+        java-version: '12'
+        distribution: 'adopt' #using a specific distribution of jdk12 (AdoptOpenJDK)
+
+    - name: Setting up Gradle
+      uses: gradle/gradle-build-action@v2
+
+    - name: Setting up Gradle Wrapper
+      run: "gradle wrapper"
+
+    - name: Validating Wrapper
+      uses: gradle/wrapper-validation-action@v1
+
+    - name: Setting up the SQLite database and Make
+      run: | 
+        sqlite3 -version
```

Comment:
Does ubuntu come with pre-installed sqlite? If not, where are we installing sqlite?

Reply:
yup, I believe it's pre-installed